devices/Button/README.md

```diff
@@ -0,0 +1,87 @@
+﻿# Button
+
+The ButtonBase class is a base implementation for buttons that is hardware independent and can be used across devices.
+The GpioButton is a GPIO implementation of the button and inherits from the ButtonBase class. This implementation has been tested on an ESP32 platform, specificaly on the [M5StickC Plus](https://shop.m5stack.com/products/m5stickc-plus-esp32-pico-mini-iot-development-kit).
+
+## Documentation
+
+Documentation for the M5StickC Plus, including pin mapping, can be [found here](https://docs.m5stack.com/en/core/m5stickc_plus).
+
+## Usage
+
+You can find an example in the [samples](./Samples/Program.cs) directory.
+
+```csharp
+// Initialize a new button
+GpioButton button = new GpioButton();
```

Comment:
Once you'll remove the default pin, don't forget to adjust the documentation.

Reply:
Thanks, fixed.